schema/types/isolator_test.go

```diff
@@ -202,8 +202,16 @@ func TestIsolatorsGetByName(t *testing.T) {
 			var r Resource = v
 			glimit := r.Limit()
 			grequest := r.Request()
-			if glimit.Value() != tt.wlimit || grequest.Value() != tt.wrequest {
-				t.Errorf("#%d: glimit=%v, want %v, grequest=%v, want %v", i, glimit.Value(), tt.wlimit, grequest.Value(), tt.wrequest)
+
+			var vlimit, vrequest int64
+			if tt.name == "resource/cpu" {
+				vlimit, vrequest = glimit.MilliValue(), grequest.MilliValue()
+			} else {
+				vlimit, vrequest = glimit.Value(), grequest.Value()
+			}
+
+			if vlimit != tt.wlimit || vrequest != tt.wrequest {
```

Comment:
I know you didn't introduce this but would you mind splitting this into two checks?